apps/examples/elf/wifiapp/Makefile

```diff
@@ -92,7 +92,7 @@ BIN_VER = 20190412
 DYNAMIC_RAM_SIZE = 51200
 KERNEL_VER = 2.0
 
-SRCS = wifiapp.c
+SRCS = $(wildcard *.c)
```

Comment:
I think better to include recovery.c only if EXAMPLES_RECOVERY_TEST is enabled ?